cabal-install/Distribution/Client/Setup.hs

```diff
@@ -1293,9 +1293,9 @@ upgradeCommand = configureCommand {
   }
 
 cleanCommand :: CommandUI CleanFlags
-cleanCommand = Cabal.cleanCommand 
```

Comment:
In the future, please try to put whitespace changes into a separate commit.

Reply:
@23Skidoo Just curious. Is there any particular reason to do them in separate commit?

Reply:
Makes life easier for the reviewer.

Reply:
@23Skidoo GitHub supports hiding of whitespace changes. You can add `?w=1` manually to the end of the link to hide whitespace changes. Like this:

* https://github.com/haskell/cabal/pull/5527/files?w=1

Oh you can open GitHub settings

<img width="717" alt="screen shot 2018-08-16 at 9 56 15 am" src="https://user-images.githubusercontent.com/4276606/44182809-1d35af00-a13b-11e8-846b-8e770df439cf.png">

But it means that reviewer still needs to do something to make his own life easier. So it's a tradeoff between who does less work. I'd prefer to do less 🙂 But since you're maintainer, it's okay for me to follow this commit style guide.